pkgs/development/libraries/cctag/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, cmake
+, boost
+, eigen
+, opencv
+, tbb
+
+, avx2Support ? stdenv.hostPlatform.avx2Support
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cctag";
+  version = "1.0.3";
+
+  outputs = [ "lib" "dev" "out" ];
```

Comment:
```suggestion
  outputs = [ "out" "lib" "dev" ];
```

Reply:
Is that really a good idea? `cctag.out` is empty (it's only included because some hooks complain otherwise). `cctag.lib` really is the main binary output.
If `cctag.lib` were to go first, building cctag would default to an empty output which would be somewhat counterintuitive.
Not sure if there are any policies surrounding this other than "binaries should go first"